app/controllers/users_controller.rb

```diff
@@ -45,6 +46,42 @@ def index
     end
   end
 
+  def show
```

Comment:
I don't think that the general-purpose "show" is a good place to expose such detailed, specific (and partially private!) information.
You're making changes to `users_controller`, which is "the global" way of accessing users. If you're making a view for merging, maybe something like `show_for_merge`? Or some other name, at least as long as it's not general-purpose.